packages/paima-sdk/paima-precompiles/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@paima/precompiles",
```

Comment:
why did this need to be a standalone package? (maybe this is the right way to do it. Not sure the rationale)

Reply:
I had added it initially as a part of the utils package, but then the `precompiles.cjs` generated in the `packaged` directory in the game contained a lot of useless code from that package. I then changed it to this in order to reduce the size of that.

Reply:
There may be a way of eliminating that code though?

Reply:
Makes sense. Usually tree-shaking should get rid of unused stuff for production builds, but in my experience tree shaking is pretty unreliable to remove deadcode. We should at least document this rationale somewhere (possibly in the README for the project)